modules/Pushsafer/index.js

```diff
@@ -0,0 +1,91 @@
+/*** Z-Wave-Me-pushsafer Z-Way HA module *******************************************
+
+Version: 1.0.5
+(c) 2023 Appzer.de
+-----------------------------------------------------------------------------
+Author: Kevin Siml
+Description: Send Push Notification by Pushsaver.com Service
+******************************************************************************/
+
+// ----------------------------------------------------------------------------
+// --- Class definition, inheritance and setup
+// ----------------------------------------------------------------------------
+
+function NotificationPushsafer (id, controller) {
+    // Call superconstructor first (AutomationModule)
+    NotificationPushsafer.super_.call(this, id, controller);
+}
+
+inherits(NotificationPushsafer, AutomationModule);
+
+_module = NotificationPushsafer;
+
+// ----------------------------------------------------------------------------
+// --- Module instance initialized
+// ----------------------------------------------------------------------------
+
+NotificationPushsafer.prototype.init = function (config) {
+    NotificationPushsafer.super_.prototype.init.call(this, config);
+
+    var self = this;
+
```

Comment:
Would be also nice if your app could provide a notification channel to allow post events thru notification filtering.

Create the channel: https://github.com/Z-Wave-Me/home-automation/blob/master/modules/NotificationChannelEmail/index.js#L78

Unregister the channel: https://github.com/Z-Wave-Me/home-automation/blob/master/modules/NotificationChannelEmail/index.js#L106